app/form_models/jobseeker_sign_in_form.rb

```diff
@@ -0,0 +1,30 @@
+class JobseekerSignInForm
+  include ActiveModel::Model
+
+  AUTHENTICATION_FAILURE_MESSAGES = [
```

Comment:
This could just be:

`AUTHENTICATION_FAILURE_MESSAGES = %w[invalid not_found_in_database locked last_attempt].map { |error| I18n.t("devise.failure.#{error}" }`